examples/resources/leanspace_streams.tf

```diff
@@ -19,7 +19,11 @@ resource "leanspace_streams" "stream" {
         type           = "FIELD"
         data_type      = "UINTEGER"
         name           = "version"
-        length_in_bits = 8
+        length {
+          unit = "BITS"
+          type = "FIXED"
+          value  =8
+        }
```

Comment:
Had another look and noticed here the formatting is messed up, btw

Reply:
Thanks, I reformatted :)